app/api/translation.py

```
@@ -0,0 +1,231 @@
+"""
+Translation API endpoints for LocPlat service.
+"""
+from typing import Dict, List, Optional, Any
+from fastapi import APIRouter, HTTPException, Depends
```

Ruff (0.11.9)

`fastapi.Depends` imported but unused

Remove unused import: `fastapi.Depends`

(F401)